dietpi/dietpi-software

```diff
@@ -970,6 +970,12 @@ Available commands:
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/social/#wordpress'
 		aSOFTWARE_DEPS[$software_id]='88 89 webserver'
 		#------------------
+		software_id=206
+		aSOFTWARE_NAME[$software_id]='Homer'
+		aSOFTWARE_DESC[$software_id]='A homepage for your services'
+		aSOFTWARE_CATX[$software_id]=6
+		aSOFTWARE_DOCS[$software_id]='https://github.com/bastienwirtz/homer'
+		#------------------
```

Comment:
You would need to add web server dependency like

```
aSOFTWARE_DEPS[$software_id]='webserver''
```

Does it require anything like PHP?

Reply:
Nope, it is all a static page. Sadly currently webservers are pulling PHP as dependency 🤔. I wanted to de-tangle this anyway, can do this after this PR has been merged.